Dockerfile

```
@@ -4,17 +4,14 @@ LABEL "maintainer"="Sviatoslav Sydorenko <wk+re-actors@sydorenko.org.ua>"
 LABEL "repository"="https://github.com/re-actors/gh-action-pypi-publish"
 LABEL "homepage"="https://github.com/re-actors/gh-action-pypi-publish"
 
-LABEL "com.github.actions.name"="pypi-publish"
-LABEL "com.github.actions.description"="Upload Python distribution packages to PyPI"
-LABEL "com.github.actions.icon"="upload-cloud"
-LABEL "com.github.actions.color"="yellow"
-
 ENV PYTHONDONTWRITEBYTECODE 1
 ENV PYTHONUNBUFFERED 1
 
-ADD LICENSE.md /LICENSE.md
-
 RUN pip install --upgrade --no-cache-dir twine
 
-ENTRYPOINT ["twine"]
-CMD ["upload", "dist/*"]
+WORKDIR /app
+COPY ./LICENSE.md /app/
```

When using a `WORKDIR` instruction, it's like `cd` in shell scripts. In this case you can just use a relative dest.
```suggestion
COPY ./LICENSE.md .
```